src/index.spec.js

```diff
@@ -62,7 +68,7 @@ describe('format', () => {
   for (const name in formatData) {
     const datum = formatData[name]
     it(name, () => {
-      expect(format(datum)).to.equal(datum.url)
+      expect(format({...datum.out})).to.equal(datum.url)
```

Comment:
It could, if format changes behaviour in anyway in the future ... tests must not be "optimized" upon implementation knowledge of tested stuff, or we reproduce in tests the risk we try to contain by testing developments.